haskell/nixpkgs.bzl

```diff
@@ -99,6 +99,9 @@ def haskell_nixpkgs_packages(name, base_attribute_path, packages, **kwargs):
         base_repository = name,
     )
 
+def _is_nix_platform(repository_ctx):
+    return repository_ctx.execute(["bash", "-c", "type -p nix-build"]).return_code == 0
```

Comment:
This will need a justification inline as to why it's not just `repository_ctx.which`.